# actions
uint32 CREATE_NODE=1
uint32 CREATE_EDGE=2
uint32 SET_NODE=3

Header header

uint32 action

uint32 src_id
uint32 dest_id

float64 relative_rad

